### ISSUE TITLE
<figure> element CSS glitch - TABs content affecting Gallery display

### ISSUE THREAD

**Issue by mylimoproject** [elementor/elementor#8327]
## Prerequisites

- [ X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X ] The issue still exists against the latest stable version of Elementor Hello Theme

## Steps to reproduce
Activate Hello Theme and insert an image gallery into a tab content  - BUG=>frontend will display gallery as a stacked list and ignore default setup and no way of adding custom CSS to fix it as per my attempts to do so.
![Hello-Theme](https://user-images.githubusercontent.com/29313121/59458734-10c8c400-8de9-11e9-9788-ed528adbe27f.JPG)
![twenty-seventeen](https://user-images.githubusercontent.com/29313121/59458735-10c8c400-8de9-11e9-99d6-4ae1746a6768.JPG)

Now activate twenty-Seventeen and see gallery displayed properly ;-) (as per screen-shot above)

## Isolating the problem

- [x ] This bug happens with only Elementor Hello Theme active (and Elementor Pro).
- [ ] This bug happens with a default WordPress theme active.
- [x ] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by MichaelDarko** [comment]
I have similar problem when i add a caption to a gallery image. Without a caption the image is formated as it should be. But when you add a caption suddenly the "figure" styling from Elementor is applied to it and it makes a mess.

I think it's this code: 
`.elementor .elementor-widget:not(.elementor-widget-text-editor) figure`

**Comment by Aviad-Herman** [comment]
Hi,
I'm merging this issue with #2587.
Currently the tabs widget is not intended to support this king of behavior.
We do have future plans for this and even more advanced behavior.

**Comment by nicholaszein** [comment]
Hello there @mylimoproject!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Core v3.10.0! 🥳 

You can now use the new **Nested Tabs** element to add Elementor widgets inside, including the gallery widget. This should solve your issue. You can learn more about it [here](https://elementor.com/help/tabs-widget-2/).

Thank you for your feedback and patience! 🙏 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
